chrome/content/zotero/xpcom/date.js

```diff
@@ -477,6 +477,48 @@ Zotero.Date = new function(){
 		return date;
 	}
 	
+	this.isHTTPDate = function(str) {
+		var dayNames = ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'];
+		var monthNames = this.getMonths().short;
```

Comment:
This gets localized dates. We need the English abbreviations from the spec.

(But not sure why you didn't just find a regexp for this whole function? E.g., http://stackoverflow.com/q/9352003)